docs/content/4.api/3.configuration.md

```diff
@@ -410,3 +410,10 @@ Toggles the document-driven mode.
 | `layoutFallbacks` |     `string[]`      | A list of `globals` key to use to find a layout fallback.      |
 | `injectPage`      |      `boolean`      | Inject `[...slug].vue` pre-configured page                     |
 | `trailingSlash`   |      `boolean`      | Add a slash at the end of  `canonical` and `og:url`            |
+
+## `respectPathCase`
```

Comment:
Maybe you could provide an example to help dev to understand types of issues that this options can solve

Reply:
Added!

Reply:
Perfect 👌